templates/product/details.html

```diff
@@ -66,7 +66,7 @@
             </div>
             <div class="col-md-6 col-12 product__info">
                 <h1 itemprop="name" class="product__info__name">
-                    {{ product }}
+                    {{ product.translated.name }}
```

Comment:
Can't we pass `product.translated` to templates? I'm not convinced all of the templates need to know about translations.